ecosystem/manager.py

```diff
@@ -80,13 +84,41 @@ def dispatch_check_workflow(
 
     def get_projects_by_tier(self, tier: str) -> None:
         """Return projects by tier.
-
         Args:
             tier: tier of ecosystem
         """
         repositories = ",".join([repo.url for repo in self.dao.get_repos_by_tier(tier)])
         set_actions_output([("repositories", repositories)])
 
+    def update_badges(self):
+        """Updates badges for projects."""
+        current_dir = os.path.dirname(os.path.abspath(__file__))
+        badges_folder_path = "{}/../badges".format(current_dir)
+
+        tests = []
+        community_projects = [
+            repo.name for repo in self.dao.get_repos_by_tier("COMMUNITY")
+        ]
+        tests_list = [
+            repo.tests_results for repo in self.dao.get_repos_by_tier("COMMUNITY")
+        ]
+
+        for i in tests_list:
+            test_result = [str(repo.passed) for repo in i]
+            if "False" not in test_result:
+                tests.append(True)
+            else:
+                tests.append(False)
+
+        for project, tests_out in zip(community_projects, tests):
```

Comment:
as I mentioned above we do not need to call db 2 times. We can just do something like:
```python
for project in self.dao.get_repos_by_tier(tier):
    tests_passed = all([result.passed for result in project.test_results])
```